config/puma.rb

```diff
@@ -38,3 +38,8 @@
 # end
 
 bind "unix:///var/run/puma/#{SOCKET_NAME}"
+
+stdout_redirect(
+  "/srv/velum/log/#{ENV.fetch("RAILS_ENV")}.log",
+  "/srv/velum/log/#{ENV.fetch("RAILS_ENV")}.log"
```

Comment:
Maybe we can make this one `"/srv/velum/log/#{ENV.fetch("RAILS_ENV")}.err"` ?

Reply:
updated